hlint-run/README.md

```diff
@@ -40,15 +40,15 @@ jobs:
   hlint:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
 
     - name: 'Set up HLint'
-      uses: haskell/actions/hlint-setup@v1
+      uses: haskell/actions/hlint-setup@v2
```

Comment:
presuming that this PR leads to a major version bump.

the reason this should probably lead cause a major version bump is that running on node 16 requires a certain minimum version of the actions runner. Some people in corporate environments still have ancient runners, so it's better to give them a safe path to upgrade.

Reply:
👍 works for me, and that's a good reason to bump.